packages/teleport/src/components/Layout/Layout.jsx

```diff
@@ -40,7 +40,7 @@ FeatureHeader.defaultProps = {
  * Header Title
  */
 const FeatureHeaderTitle = styled(Text)`
-  flex-shrink: 0;
+  white-space: nowrap;
```

Comment:
`flex-shrink: 0` prevented ellipsis from working b/c flexed containers wouldn't shrink.

I think we use flex shrink 0 here to prevent titles from wrapping, so `white-space: no wrap` achieved the same behavior.

Reply:
`flex-shrink:0` is used to prevent strange squeezing behavior of this container in Safari. Try out minimizing the window height when a page has some content in it (populated table for many rows for example). Maybe we can try minHeight instead.

Reply:
I think i'm blind, I'm not able to reproduce the behavior in safari... I tried by removing both flex-shrink and white-space. I'm thinking it got fixed elsewhere? I still need the `white-space: nowrap` to prevent title wrapping though.